pkg/tools/topo-generator/parser/convert.go

```diff
@@ -0,0 +1,152 @@
+package parser
+
+import (
+	"strings"
+	"regexp"
+	"strconv"
+	"github.com/google/uuid"
+	"github.com/onosproject/onos-topo/pkg/tools/topo-generator/reader"
+)
+
+type Underlay struct {
+	Networks []Network 
+}
+
+type Network struct {
+    Entity_Id       string      
+	Name			string		
+    Display_Name    string  
```

Comment:
Perhaps github shows the attributes are not aligned. If that is not the case make them alight. IF you use Goland, it does it automatically.

Reply:
In VSCode they show up as aligned, I am not sure why they are unaligned here. I can fix it manually.